jsconfig.json

```diff
@@ -12,7 +12,7 @@
       "@Types/*": ["src/redux/actions/types/*"],
       "@Common/*": ["src/components/common/*"],
       "@Views/*": ["src/components/views/*"],
-      "@Utils/*": ["utils/*"],
+      "@Utils/*": ["utils/*"]
```

Comment:
Why did you take out the trailing comma here?

Reply:
My eslint was flagging it.